source/configuration/write.txt

```diff
@@ -72,11 +72,11 @@ The following options for writing to MongoDB are available:
 
        - ``insert``: insert the data.
        - ``replace``: replace an existing document that matches the
-         ``idFieldList`` value with the new data, or insert the data if
-         no match exists.
+         ``idFieldList`` value with the new data. If no match exists, the
+         behavior depends on ``upsertDocument``.
```

Comment:
This behavior seems tough to describe more clearly, but maybe by specifying how it depends on "upsertDocument" would make it more clear.

```suggestion
         ``idFieldList`` value with the new data. If no match exists, the
         connector uses the value of  the ``upsertDocument``setting
         to determine whether to perform an upsert.
```

Similar comment for lines 78-79.